mithril-test-lab/mithril-devnet/devnet-mkfiles.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 # Debug mode
-if [ -v DEBUG ]; then
+if [ -n $DEBUG ]; then
```

Comment:
I think you should add two brackets more.
```suggestion
if [[ -n $DEBUG ]]; then
```